src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$second-color: #616070;
+
+body,
+p {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  width: 200px;
+  align-items: center;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    padding-top: 32px;
+    padding-left: 19px;
+    padding-right: 19px;
+    margin-bottom: 40px;
+  }
+  &__name {
+    width: 166px;
+    height: 36px;
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0;
+    margin-bottom: 4px;
+    color: $main-color;
+  }
+  &__code {
+    width: 98px;
+    height: 14px;
+    padding-left: 16px;
+    padding-right: 84px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $second-color;
+  }
+  &__price {
+    display: flex;
+    align-items: center;
+    font-family: $font-family;
+    width: 166px;
+    height: 18px;
+    margin-bottom: 16px;
+    justify-content: space-between;
+  }
+  &__price-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    width: 31px;
+    height: 18px;
+    color: $second-color;
+    padding-right: 18px;
+  }
+  &__price-value {
+    width: 50px;
+    height: 18px;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+  &__button {
+    width: 166px;
+    height: 40px;
+    padding: 0;
+    border: hidden;
```

Comment:
The `border: hidden;` declaration is not standard. If you want to hide the border, you should use `border: none;` or `border: 0;` instead.